# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/ContentChild.h
# Commit: 312b4abacc2d
# Full Hash: 312b4abacc2dfe0ab2014d62ca271ac54af6704d
# Author: Dimi Lee <dlee@mozilla.com>
# Date: 2020-04-23 21:43:09
# Regressor Bug: 1616775
# File Overlap Count: 1
# Description:
#   Bug 1616775 - P7. Add IPC for ContentBlockingNotifier::OnDecision r=timhuang,baku
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D71016
# ==============================================================================

diff -r 9be5ae499d84 -r 312b4abacc2d dom/ipc/ContentChild.h
--- a/dom/ipc/ContentChild.h	Thu Apr 23 14:25:31 2020 +0000
+++ b/dom/ipc/ContentChild.h	Thu Apr 23 14:25:33 2020 +0000
@@ -678,6 +678,11 @@
       const nsCString& aTrackingOrigin, uint32_t aCookieBehavior,
       const ContentBlockingNotifier::StorageAccessGrantedReason& aReason);
 
+  mozilla::ipc::IPCResult RecvOnContentBlockingDecision(
+      const MaybeDiscarded<BrowsingContext>& aContext,
+      const ContentBlockingNotifier::BlockingDecision& aDecision,
+      uint32_t aRejectedReason);
+
 #ifdef NIGHTLY_BUILD
   // Fetch the current number of pending input events.
   //